common/app/views/fragments/exploreSeries.scala.html

```diff
@@ -54,7 +54,7 @@
                 }
             }
         </div>
-        @if(!page.item.elements.hasMainVideo){
+        @if(!page.item.elements.hasMainVideo && !page.item.elements.hasMainEmbed){
```

Comment:
Similarly, we can replace `page.item.elements.hasMainVideo && !page.item.elements.hasMainEmbed` with `hasMainMedia && ! hasEmbed`